src/interface/cron/events/process.js

```diff
@@ -2,15 +2,18 @@ const { EventProcessor, Event } = require("../../../infra/database/models");
 const agenda = require("../index");
 const jobs = require("../jobs");
 const processEvent = require('./processEvent');
-const FetchEventCount = 10;
-const { EVENT_PROCESS_MAX_RETRIES } = require("../../../constants");
+const constants = require("../../../constants");
+
+const FetchEventCount = constants.CRON.PROCESS_LIMIT;
 
 agenda.define(jobs.PROCESS, async (job, done) => {
   try {
-    const minCheckpoint = await fetchMinCheckpoint();
-    const events = await fetchEvents(minCheckpoint, FetchEventCount);
-    console.log("Received entries", jobs.PROCESS, events.length);
-    await processStoredEvents(events);
+    const checkpoints = await fetchCheckspoints();
+    for (const checkpoint of checkpoints) {

```

Comment:
Wont this make redundant requests to db in case some checkpoints are already processed ?